src/lib/components/progress/QCircularProgress.svelte

```diff
@@ -18,23 +18,34 @@
     quaffClasses: [color && `text-${color}`],
     userClasses,
   });
+
+  $: circumference = 2 * Math.PI * 20;
```

Comment:
This shouldn't be a reactive value, it should be a `const`. Or maybe you wanted it to change with `size`?